app/controllers/filler/pokemon.js

```diff
@@ -7,29 +7,21 @@ const PokemonModel = require(__appbase + 'models/pokemon'),
 module.exports = {
     fill: function (callback) {
 
-        var afterInsertion = function () {
-            logger.info('Pokemon Insertion Completed');
-            callback(false);
-        };
 
-        var file = __tmpbase + 'pokemon.json';
+    },
+    insertToDb: function (callback) {
+        logger.info('MongoDb Insertion...');
+        const file = __tmpbase + 'pokemon.json';
 
-        jsonfile.readFile(file, function (err, obj) {
-            if (obj !== undefined) {
-                module.exports.insertToDb(obj, afterInsertion);
+        jsonfile.readFile(file, function (err, pokemons) {
+            if (pokemons !== undefined) {
+                insert(pokemons);
             }
         });
-    },
-    insertToDb: function (pokemons, callback) {
-        logger.info('MongoDb Insertion...');
 
         var addPokemon = function (pokemon, callback) {
             PokemonStore.add(pokemon, function (success, data) {
-                if (success != 1) {
-                    logger.success('Success: ' + data.pokemonId);
-                } else {
-                    logger.error('Error:' + data);
-                }
+                logger.success((success != 1) ? 'Error:' + data : 'Success: ' + data.pokemonId);
```

Comment:
This will cause success message to be logged in either case... 
It is better to revert this change.